cognite/client/_api_client.py

```diff
@@ -336,12 +342,51 @@ def _list(
             limit=limit,
             chunk_size=self._LIST_LIMIT,
             filter=filter,
-            headers=headers,
             other_params=other_params,
+            headers=headers,
         ):
             items.extend(resource_list.data)
         return cls(items, cognite_client=self._cognite_client)
 
+    def _list_partitioned(
```

Comment:
Should have its own unit tests in tests_unit/test_api_client.py where you should test that correct bodies are sent to API and also test cases where some calls fail.